### ISSUE TITLE
Cursor does this weird dance while scrolling

### ISSUE THREAD

**Issue by Nostang3** [puregallery/Pure-Gallery-Support#94]
I have 7 albums in a 2x2 view and when shortly after the halfway point it will jump back a bit and vice versa in the other direction. It looks to be the point where it is loading the 7th album alone at the bottom when it happens or when it loads the top two albums when scrolling up.

**Comment by puregallery** [comment]
by cursor...do you mean the scrollbar? Or does the list itself jump/scroll weirdly?

**Comment by Nostang3** [comment]
Yup. Again, I need to English better. The scroll bar will go from like say 60% back to 50% and you're scrolling down.

**Comment by puregallery** [comment]
No worries.
I will look into it.

**Comment by puregallery** [comment]
This is a bug in Android's implementation of RecyclerView...but in the normal state, they compensate it by changing the size of scrollbar so that the jump is not so annoying.
I'll see what I can do about this...
